internal/zinc-scripted/src/test/scala/sbt/internal/inc/IncHandler.scala

```diff
@@ -214,6 +225,9 @@ case class ProjectStructure(
     lookupProject: String => ProjectStructure,
     scalaVersion: String
 ) extends BridgeProviderSpecification {
+
+  val useStraightToJar = false
```

Comment:
Let's add a flag so that we can test this in the CI. I propose to enrich the interface of zinc scripted with a parameter that, when passed, will run all tests with straight-to-jar compilation enabled.

Reply:
I just implemented that in form of `zincScripted/test:run --to-jar`. For runs from sbt I added similar flag to `scriptedBufferLog`, but perhaps I should add a new task like `scriptedToJar`? But then also `scriptedUnpublishedToJar`? What do you think?

Reply:
I think it's not necessary to add new tasks :+1: